mlrun/feature_store/api.py

```diff
@@ -117,7 +123,8 @@ def get_offline_features(
     :param end_time:        datetime, high limit of time needed to be filtered. Optional.
         entity_timestamp_column must be passed when using time filtering.
     """
-    feature_vector = _features_to_vector(feature_vector)
+
+    feature_vector = _features_to_vector_and_check_permissions(feature_vector)
```

Comment:
do we also need to check read permissions on each feature set the vector is built from?

Reply:
this is done in get_feature_set_by_uri